win32ss/include/callback.h

```diff
@@ -102,6 +102,15 @@ typedef struct _EVENTPROC_CALLBACK_ARGUMENTS
   INT_PTR Mod;
   ULONG_PTR offPfn;
 } EVENTPROC_CALLBACK_ARGUMENTS, *PEVENTPROC_CALLBACK_ARGUMENTS;
+ 
+typedef struct _LOADIMAGE_CALLBACK_ARGUMENTS
+{
+    WCHAR ImageName[MAX_PATH];
+    UINT ImageType;
+    int cxDesired;
+    int cyDesired;
+    UINT fuFlags;
```

Comment:
perhaps ImageName can be moved at the end of this struct and make it 0 sized. When allocating this struct you can add the size of the string so as to save some space in the stack.